tests/integration/data/regression_output/linux/dmesg^$^pp.output

```diff
@@ -0,0 +1,1267 @@
+[    0.000000]: Linux version 5.0.0-36-generic (buildd@lgw01-amd64-060) (gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)) #39~18.04.1-Ubuntu SMP Tue Nov 12 11:09:50 UTC 2019 (Ubuntu 5.0.0-36.39~18.04.1-generic 5.0.21)
```

Comment:
did you consider using compressed output files?

that'd mean we'd have to decompress, recompress when making changes.. and also decompress when running the tests.. but I wonder if that'd be worth the trade off of saving disk space by the repository.. I don't feel strongly either way.

Reply:
I'd be against that.

The files don't take a lot of space and the big upside is that we can actually review them in PRs when they change here in Github because they are human-readable.